src/js/components/Carousel.js

```diff
@@ -303,7 +318,8 @@ export default class Carousel extends Component {
       <div ref={ref => this.carouselRef = ref} className={classes}
         role='group' aria-label={carouselMessage}
         onFocus={this._stopAutoplay} onBlur={this._startAutoplay}
-        onMouseOver={this._stopAutoplay} onMouseOut={this._startAutoplay}>
+        onMouseOver={this._stopAutoplay} onMouseOut={this._startAutoplay}
+        {...DOMProps}>
```

Comment:
I think we need to put the DOMProps right after the `ref` property so that the Carousel properties don't get overridden.